package.json

```diff
@@ -2,8 +2,20 @@
   "name": "is-dom-node",
   "version": "1.0.0",
   "description": "TypeScript helper functions to narrow the type of object to a Node descendant",
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
That's a good question. I've never used one and it seems that TypeScript will check for `.d.ts` files in the correct folder. I've already imported this code to the `xml-crypto` project to make sure that it all works and I didn't have any trouble with types there. I haven't looked super hard, but I found one reference that says that TypeScript will look in the correct location and another that says you should add it. IDK.

Honestly, for so many projects with lots of `.d.ts` files, I feel like TypeScript must be doing the right thing because I don't see a lot of times (any?) where all the types are exported to a single `.d.ts` file.

Reply:
Ok, good enough for now.
Once we publish to npm I woupd like to check that it indicates that the package comes with types.